src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -23,15 +23,21 @@ class SmsMessage implements MessageInterface
     private ?string $transport = null;
     private string $subject;
     private string $phone;
+    private ?string $from;
 
-    public function __construct(string $phone, string $subject)
+    public function __construct(string $phone, string $subject, string $from = null)
```

Comment:
```suggestion
    public function __construct(string $phone, string $subject, string $from = '')
```

Reply:
@fabpot 

I'm not sure to understand why is it better ?

In transports we will have to do `$from = '' !== $message->getFrom() ? $message->getFrom() : $this->from;` instead of `$from = $message->getFrom() ?? $this->from;` Seems to be less clear

Reply:
I don't see the need to have `null` in addition to an empty string.

`$from = $message->getFrom() ? $message->getFrom() : $this->from` is enough.